source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -1068,6 +1100,37 @@ void ESolver_KS_LCAO<TK, TR>::iter_finish(int& iter)
     {
         GlobalC::dftu.initialed_locale = true;
     }
+
+    // 7) rdmft, added by jghan, 2024-10-25
+    if ( PARAM.inp.ab_initio_type == "rdmft" )
```

Comment:
leaving approximately 5 lines here in esolver, I would suggest not adding any detailed operations in esolver_ks_lcao, the file is already very big.